packages/core/src/git.ts

```diff
@@ -69,6 +72,7 @@ export default class Git {
       .plugin(throttling);
     this.github = new gitHub({
       baseUrl: this.baseUrl,
+      agent: proxyUrl ? new HttpsProxyAgent(proxyUrl) : undefined,
```

Comment:
We're setting an agent into this constructor through `options`, but I don't see it being used anywhere. 
Instead of creating a new `agent`, can we just use the value of `this.options.agent`?

Reply:
or better yet, just pass in the `proxy` in the options, and construct the `HttpsProxyAgent` here using that passed in `proxy`

Reply:
modified the code to use the agent and pass to git.